src/rnpkeys/rnpkeys.cpp

```diff
@@ -230,15 +230,25 @@ bool
 rnp_cmd(rnp_cfg_t *cfg, cli_rnp_t *rnp, optdefs_t cmd, const char *f)
 {
     const char *key;
+    std::string fs;
 
     switch (cmd) {
     case CMD_LIST_KEYS:
         if (!f) {
-            f = rnp_cfg_getstr(cfg, CFG_USERID);
+            list *ids = NULL;
+            if ((ids = rnp_cfg_getlist(cfg, CFG_USERID)) && list_length(*ids) > 0) {
+                f = (fs = rnp_cfg_getlist_string(cfg, CFG_USERID, 0)).c_str();
```

Comment:
I missed it during the previous review - if you don't use fs later on then it would be enough to write `f = rnp_cfg_getlist_string(cfg, CFG_USERID, 0).c_str()`.

Reply:
I believe the fs used to keep values of strings, pointers from c_str() won't work without the fs.

Reply:
Oh, agreed. Missed this side, sorry.